graviti/dataframe/frame.py

```diff
@@ -142,6 +129,20 @@ def _construct(
         obj._index = index
         return obj
 
+    @classmethod
```

Comment:
The commit message is wield too
here is an reference:
`support reading data from TensorBay to local DataFrame`?